src/client/renderer/r_context.c

```diff
@@ -264,10 +264,12 @@ void R_InitContext(void) {
 
 	Com_Print("  Trying %dx%d..\n", w, h);
 
-	SDL_GL_SetAttribute(SDL_GL_RED_SIZE, 8);
-	SDL_GL_SetAttribute(SDL_GL_GREEN_SIZE, 8);
-	SDL_GL_SetAttribute(SDL_GL_BLUE_SIZE, 8);
-	SDL_GL_SetAttribute(SDL_GL_ALPHA_SIZE, 8);
+	SDL_GL_SetAttribute(SDL_GL_FRAMEBUFFER_SRGB_CAPABLE, 1);
+
+	SDL_GL_SetAttribute(SDL_GL_RED_SIZE, 32);
```

Comment:
Is there any merit to using full 32 bit floats here? Would we see a difference with 16 bit "half" floats?